windows_containers/windows-containers-release-notes-8-x.adoc

```diff
@@ -33,6 +33,11 @@ endif::openshift-origin[]
 
 This release of the WMCO provides new features and bug fixes for running Windows compute nodes in an {product-title} cluster. The components of the WMCO 8.0.0 were released in link:https://access.redhat.com/errata/RHSA-2023:1372[RHSA-2023:1372].
 
+[IMPORTANT]
+====
+Due to a known issue, the WMCO 8.0.0 is not available for download and use. The issue will be addressed in WMCO 8.0.1, which is planned for release and will be available for download soon. If you upgrade your cluster from {product-title} 4.12 to {product-title} 4.13, you can continue to use WMCO 7.0.x. However, you will not be able to use the new WMCO 8.0.0 functionality, as described in this section.
```

Comment:
```suggestion
Due to a known issue, the WMCO 8.0.0 release is not available to download and use. The issue will be addressed in WMCO 8.0.1, which is planned for release. If you upgrade your cluster from {product-title} 4.12 to {product-title} 4.13, you can continue to use WMCO 7.0.x. However, you will not be able to use the new WMCO 8.0.0 functionality, as described in this section.
```
- You either need a noun after 8.0.0 or to remove "the."
- It's better to not use a word as both a noun and a verb, especially in the same note.
- Unless you can say with more detail when it's going to be released, I think it's better to avoid a timeline.